modules/webgl/src/classes/buffer.js

```diff
@@ -93,7 +93,7 @@ export default class Buffer extends Resource {
 
     // Set data: (re)initializes the buffer
     if (props.data) {
-      this._setData(props.data);
+      this._setData(props.data, props.offset, props.size);
```

Comment:
Doesn't `size` refer to the attribute size in the accessor? Should this be called `props.byteLength`?

Reply:
Done.